packages/Card/src/Card.js

```diff
@@ -38,9 +38,9 @@ function Card(props) {
   };
 
   return (
-    <sc.cardStyles data-pka-anchor="card" {...styleProps} {...moreProps}>
+    <sc.CardWrapper data-pka-anchor="card" {...styleProps} {...moreProps}>
```

Comment:
I'm not a fan of the `Wrapper` suffix for naming in general – it's too vague to be helpful most of the time – but sometimes it really _is_ necessary.  But in this case I think simply `<sc.Card>` fits nicely, is a bit less typing, and a bit easier to read (will, at least when the component name also gets long).  

Bit of a hassle to make this adjustment throughout, but I think it's worth it to get it done right and then move on.  Thanks!